index.js

```diff
@@ -73,16 +75,26 @@ function writeDescription (schema, options) {
         : typeMatch || schema.type
       }}`
   }
-  return `${indent(options)} *${description}
-${indent(options)} * @${options.objectTagName || 'typedef'}${type}${schema.title
+  let descriptionLine = ''
+  if (description || options.emptyDescriptionLineBreak !== false) {
+    descriptionLine = `${indent(options)} *${description}
+`
+  }
+  return `${descriptionLine}${indent(options)} * @${options.objectTagName || 'typedef'}${type}${schema.title
     ? ` ${options.capitalizeTitle === false ? schema.title : upperFirst(schema.title)}`
     : ''
   }
 `
 }
 
-function writeProperty (type, field, description = '', optional, options) {
-  const fieldTemplate = optional ? `[${field}]` : field
+function writeProperty (type, field, description = '', optional, deflt, options) {
```

Comment:
I would use `defaultValue` for readability

Reply:
Sure, amended.